### ISSUE TITLE
<relationEntry>

### ISSUE THREAD

**Issue by SJagodzinski** [SAA-SDT/eac-cpf-schema#218]
## Relation Entry

- remove the element from EAC and replace by `<targetEntity>` #215 

## Creator of issue
1. Silke Jagodzinski
2. TS-EAS: EAC-CPF subgroup
3. silkejagodzinski@gmail.com

## Related issues / documents
[Paper on Relation](https://github.com/SAA-SDT/TS-EAS-subteam-notes/tree/master/eaccpf-subteam/working-documents/topics/relations)

## EAD3 Reconciliation
- availible in EAD 

## Context
A descriptive element for relations that identifies the relationship in a textual form.

`<relationEntry>` contains the name of the related entity. `<targetEntity>` is a more distinct element name and the new specification can encode the related entity with literals //targetEntity/part or via URI representing the literal (value).

**Comment by kerstarno** [comment]
Tested as part of the Schema Team's schema testing:

- `<relationEntry>` is not used anymore
- `<targetEntity>` is a required sub-element of `<relation>` (which will replace `<cpfRelation>`, `<functionRelation>`, and `<resourceRelation>`)
- `<targetEntity>` cannot be repeated
- `<targetEntity>` requires the sub-element `<part>`
- `<targetEntity>` requires the attribute `@targetType`
- `<targetEntity>` allows for the optional attributes `@audience`, `@id`, `@target`, `@languageOfElement`, `@scriptOfElement`, `@conventionDeclarationReference`, `@maintenanceEventReference`, `@sourceReference`, `@valueURI`, `@vocabularySource`, and `@vocabularySourceURI`
  - `<targetEntity>` does currently not allow for the optional attributes `@localType` and `@localTypeDeclarationReference`, though `<relationEntry>` in EAC-CPF 1.0 does have `@localType`
  - @SJagodzinski and @fordmadox I've tried verifying this via the relations topic paper, but the latest version of this document doesn't reflect the latest status in terms of the various attributes (e.g. doesn't list the language attributes, but lists `@encodingAnalog`); would we hence follow the general decision re `@localType` to keep it where it is currently used in EAC-CPF 1.0?  
- `<targetEntity>` also allows for optional attributes from other namespaces (tested with XLink)

The above applies to both schemas, RNG and XSD.

**Comment by fordmadox** [comment]
@kerstarno @SJagodzinski : I don't recall the history of conversions around the "relations paper" and the decision for where localType is allowed.  Update: ~~I think it makes sense to have localType on the new targetEntity element, though.~~ As for localType, that is allowed on "part", so we can map `relationEntry/@localType` --> `targetEntity/part/@localType` (since I don't see any instances of where multiple relationEntry elements need to be split into different relation declarations).

I also have a few questions around the migration of relations from 1.0 to 2.0, so it would be great if someone could come up with a mapping document so that we have a consistent approach and understanding.  So far, I've taken a few best guesses, but I would be happy to refine any and all of my guesses in the transformation process, of course :smile:

**Comment by kerstarno** [comment]
I've added a sample XML file to the "2020-2021" folder in the "SchemaTeam" Google Drive area, where I've used one of the `<cpfRelation>`-s of the _Frankenstein EAC-CPF 1.0_ as an example to convert this to EAC-CPF 2.0. There is also the start of a mapping table in the same folder. 

One tricky aspect: as `<relationEntry>` was repeatable, but `<targetEntity>` is not, we would need to repeat the whole `<...Relation>` (probably will all its sub-elements) in case of a repeated `<relationEntry>`.
